Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -51,6 +51,11 @@ public class ModelGroup {
       public List<IHandModel> modelsCheckedOut;
       public bool IsEnabled = true;
       public bool CanDuplicate;
+
+      [System.Serializable]
+      public class HandEvent : UnityEvent<Hand> { }
```

Comment:
Would be great to move this somewhere else (maybe inside the HandUtil.cs file?) so that it can more easily be references by other classes wanting to do the same thing.  For example there is a class InteractionHandEvents that defines the exact same event type, would be great to refactor it to use this new class you created from core.